tethysext/atcore/services/workflow_manager/condor_workflow_manager.py

```diff
@@ -200,7 +200,13 @@ def prepare(self):
         # Parametrize each job
         for job in self.jobs:
             # Set arguments for each job
-            job.set_attribute('arguments', self.job_args)
+            existing_job_args = job.get_attribute('arguments')
+            if existing_job_args and type(existing_job_args) != list:
```

Comment:
When does this case happen? I would expect the `arguments` attribute to return either a list of values or a string of values that have been concatenated from a list of values. If the latter case, we need to split the string again into a list rather than just wrapping it in a list.

In other words, this:

`['arg1, arg2']`

is not the same as this:

`['arg2', 'arg2']`

The latter example is what we are after. A good test of the expected behavior would be to test with multiple extra arguments.

Reply:
It's coming back as a string.  With multiple arguments, they are separated with a space between each argument.